prog/dftb+/prg_dftb/parser.F90

```diff
@@ -1333,8 +1333,36 @@ subroutine readDFTBHam(node, ctrl, geo, slako)
     else
       skInterMeth = skEqGridNew
     end if
-    call readSKFiles(skFiles, geo%nSpecies, slako, slako%orb, &
-        &angShells, ctrl%tOrbResolved, skInterMeth, repPoly)
+    call getChild(node, "TruncateSKRange", child, requested=.false.)
+    if (associated(child)) then
+      call warning("Artificially truncating the SK table, this is normally a bad idea!")
+      ! Artificially truncate the SK table
+      call getChildValue(child, "SKMaxDistance", truncationCutOff, -1.0_dp, modifier=modifier,&
```

Comment:
IMO, we should not have a default here. The block `TruncateSKRange` is already optional, if it has been specified, the user should provide a max distance. We could then also get rid of the misuse of `-1.0` for indicating a special choice. :wink:

Reply:
Now packaged as a subroutine without a default for SKMaxDistance